src/html.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
```

Comment:
there's a mix of single and double quotes. wonder how ESLint couldn't pick it up.

for module imports, we use single quotes everywhere: 

```js
import React from 'react';
```